aws_doc_sdk_examples_tools/metadata.py

```diff
@@ -6,6 +6,7 @@
 
 from collections import defaultdict
 from dataclasses import dataclass, field
+from .project_validator import ValidationConfig
```

Comment:
(nit) (sort) prefer local imports below global imports (I think? For consistency?)

Reply:
Done.